hedera-node/src/test/java/com/hedera/services/legacy/unit/LegacyMerkleOptionalBlobTest.java

```diff
@@ -33,11 +34,12 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class LegacyMerkleOptionalBlobTest {
+@Disabled
+class LegacyMerkleOptionalBlobTest {
```

Comment:
Please note that this `LegacyMerkleOptionalBlobTest` suite was intentionally disabled for stability, until blob operation synchronization is resolved. Manually running this test suite alone will still pass.